src/main/java/org/scoverage/plugin/SCoveragePreCompileMojo.java

```diff
@@ -457,9 +461,9 @@ private Artifact getScalaScoveragePluginArtifact( String resolvedScalaVersion, S
                     resolvedScalaVersion,  resolvedScalacPluginVersion, scalaMainVersion, resolvedScalacPluginVersion ) );
 
             // for scalac-scoverage-plugin versions up to 1.4.1
```

Comment:
this comment should be amended?

Reply:
I've modified the code to have specific cases for the versions of scalac-scoverage-plugin that were published.